bark/models/tests/dynamic_test.cc

```diff
@@ -84,26 +84,28 @@ TEST(valid_trajectory_test, dynamic_test) {
   EXPECT_FALSE(IsValid(traj4));
 }
 
+double CalcAccLat(const State& x, const State& x1, const double dt) {
+  auto theta_dot = (x1(static_cast<int>(StateDefinition::THETA_POSITION)) -
+                    x(static_cast<int>(StateDefinition::THETA_POSITION))) /
+                    dt;
+  return theta_dot * x(static_cast<int>(StateDefinition::VEL_POSITION));
```

Comment:
theta_dot [rad/s] * v [m/s]. Is that correct?

Could we not just: acceleration * cos(theta)?